City-of-Helsinki/drupal-helfi-platform-config
# UHF-5263: removed maintenance mode permission from content_producer and editor roles

## What has been done
Removed `access site in maintenance mode` permission from `content_producer` and `editor` roles.

## How to install
1. Set up any site instance
2. Checkout this branch of the module: `composer require drupal/helfi_platform_config:dev-UHF-5263_content_producer_permissions`
3. Run database changes: `make drush-updb`

## How to test
Go to the People → Permissions admin page and see that the permission is missing from the mentioned roles.
